test_requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
+mock
+decorator
+pylint
+black ; python_version > '3.6'
```

Comment:
Are you sure? Black is 3.6 compatible, so this could be `python_version >= '3.6'`